active_plugins/runcellpose.py

```diff
@@ -658,4 +658,7 @@ def upgrade_settings(self, setting_values, variable_revision_number, module_name
         if variable_revision_number == 2:
             setting_values = setting_values + ["0.0", False, "15", "1.0", False, False]
             variable_revision_number = 3
+        if variable_revision_number == 3:
+            setting_values = ["Python"]

```

Comment:
We definitely want to be adding here, not `=` ; the setting values also need to be on the order of the `settings` function!